src/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/DoWorkEvent.cs

```diff
@@ -6,49 +6,22 @@ namespace System.ComponentModel
 {
     public delegate void DoWorkEventHandler(object sender, DoWorkEventArgs e);
 
-    public class DoWorkEventArgs : EventArgs
+    public class DoWorkEventArgs : CancelEventArgs
     {
         private readonly object _argument;
         private object _result;
-        private bool _cancel;
 
         public DoWorkEventArgs(object argument)
         {
             _argument = argument;
         }
 
-        public object Argument
-        {
-            get 
-            {
-                return _argument;
-            }
-        }
+        public object Argument => _argument;
```

Comment:
While you're at it, this could be simplified further, e.g.:

```c#
public class DoWorkEventArgs : CancelEventArgs
{
    public DoWorkEventArgs(object argument)
    {
        Argument = argument;
    }

    public object Argument { get; }

    public object Result { get; set; }
}
```